pvlib/modelchain.py

```diff
@@ -354,6 +379,9 @@ def __init__(self, system, location,
         self.solar_position_method = solar_position_method
         self.airmass_model = airmass_model
 
+        # check that every array has parameters for the same models
+        self._check_consistent_params
```

Comment:
Statement has no effect. This is an excellent idea @cwhanse, should it be in `ModelChain`, though? I'm thinking it might have a place in `PVSystem`?

```suggestion
        # check that every array has parameters for the same models
        self._check_consistent_params()
```